lib/api/metric.js

```diff
@@ -110,10 +93,26 @@ function send(metric, points, extra, callback){
 function send_all(metrics, callback){
     var now = parseInt(new Date().getTime() / 1000);
     for(var i = 0; i < metrics.length; ++i){
-        if(!Array.isArray(metrics[i].points)){
-            metrics[i].points = [now, metrics[i].points];
-        }
+        // Try to normalize `points`
+        // DEV: We need `points` to be an array of arrays regardless of what they give us
+        // Always wrap points in an array, this way we will get:
+        //   500 => [500]
+        //   [<timestamp>, 500] => [[<timestamp>, 500]]
+        points = [metrics[i].points];
+        points = points.map(function(point){
+            // Make sure each point is an array, if not make array with current timestamp
+            //   500 => [<timestamp>, 500]
+            //   [<timestamp>, 500] => unchanged
+            if(!Array.isArray(point)){
+                var now = parseInt(new Date().getTime() / 1000);
+                point = [now, point];
+            }
+            return point;
```

Comment:
What kind of inputs do you want to support for `metrics[i].points`? I believe this will work for `points: 40`, `points: [<timestamp>, 40]`, and `points: [40, 32, 75]`. It will _not_ work for `points: [[<timestamp>, 40], [<other_timestamp>, 32]]`